insights_content_template_renderer/DoT.py

```diff
@@ -100,51 +98,77 @@ def _conditional(elsecode, code):
                 return "';}out+='"
 
     def _iterate(iterate, vname, iname, sid=sid):
-        if (not iterate or not vname):
+        if not iterate or not vname:
             return "';} } out+='"
 
         sid += 1
         indv = iname or "i" + str(sid)
         iterate = unescape(iterate)
 
         _sid = str(sid)
-#        print iterate, vname, iname, _sid
-
-        return "';var arr" + _sid + "=" + iterate + ";if(arr" + _sid + "){var " \
-               + vname + "," + indv + "=-1,l" + _sid + "=arr" + _sid + ".length-1;while(" \
-               + indv + "<l" + _sid + "){" + vname + "=arr" + _sid + "[" + indv + "+=1];out+='"
+        #        print iterate, vname, iname, _sid
```

Comment:
left over?

Reply:
hm... it was there already. I reformatted the file though, maybe that's why it appears as a change